### ISSUE TITLE
fftw.pc in fftw-3.3.8 declares version as 3.3.7

### ISSUE THREAD

**Issue by duncanmmacleod** [conda-forge/fftw-feedstock#62]
Issue:

The `fftw.pc` file shipped with fftw-3.3.8-nompi_hbc43ac9_1110 declares

```pc
Version: 3.3.7
```

rather than the actual packaged version...

**Comment by beckermr** [comment]
Is this issue upstream?

**Comment by duncanmmacleod** [comment]
Checking now...

**Comment by duncanmmacleod** [comment]
The tarball for 3.3.8 includes this:

```
> grep FFTW_VERSION .\CMakeLists.txt
set (FFTW_VERSION 3.3.7)
set (VERSION ${FFTW_VERSION})
```

**Comment by duncanmmacleod** [comment]
Ok, it looks like the cmake and autotools builds are not synchronised upstream...

**Comment by beckermr** [comment]
OK. So this bug report should be filed with them. 

If you'd like to submit a patch, we can patch it here as well.

**Comment by duncanmmacleod** [comment]
I think this has already been fixed upstream, but 3.3.9 hasn't been released yet. I'll patch this and backport something else that should be helpful...
